app.py

```diff
@@ -661,8 +656,6 @@ def external(attrs, new=False):
         'countries': country_data,
 
         # Context info
-        'details_api_error': details_response.old_data_from_error,
-        'metrics_api_error': metrics_response.old_data_from_error,
```

Comment:
I removed those lines because:
1. we don't actually use them
2. We handle errors in another way (see details function for example)